### PR TITLE
AP-4095: Update the existing HMRC playback pages for client

### PR DESCRIPTION
**Pull request by agoldstone93**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4095)

- Updated to match designs in ticket
- Replaced table with govuk component table
- Added custom sass to remove bottom border on one table row. This is done in [here](https://govuk-components.netlify.app/components/table/#complex-tables) using `no-border-bottom` class but I couldn't get it work without custom css

To do:
- [x] Make sure caption is in right place when there's an error on the page

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/providers/means/full_employment_details/show.html.erb [hunk 1]
```diff
@@ -2,6 +2,8 @@
               url: providers_legal_aid_application_means_full_employment_details_path(@legal_aid_application),
               method: :patch,
               local: true) do |form| %>
+
+  <span class="govuk-caption-xl"><%= t("providers.means.client_caption") %></span>
```
Comment (colinbruce) [inline app/views/providers/means/full_employment_details/show.html.erb:6]: The pages I'm working on have the caption as `govuk-caption-l`
Just checking. If this page is different, we should check with the design team!

File: config/locales/en/providers.yml [hunk 2]
```diff
@@ -819,12 +819,16 @@ en:
       employment_incomes:
         show:
           hmrc-information: HMRC have provided the information on this page.
-          inset_text: "We've used your client's National Insurance number to get this information from HMRC."
+          inset_text: We used your client’s National Insurance number to get this information from HMRC.
```
Comment (colinbruce) [inline config/locales/en/providers.yml]: The apostrophe here is different to the one on line 828

**Review by colinbruce — APPROVED** [review]
A couple of minor observations, but otherwise LGTM :+1:

File: config/locales/en/providers.yml (hunk 2 above)
Comment (agoldstone93) [reply config/locales/en/providers.yml]: This is already fixed in a later commit. Good spot though!

File: app/views/providers/means/full_employment_details/show.html.erb (hunk 1 above)
Comment (agoldstone93) [reply app/views/providers/means/full_employment_details/show.html.erb]: Good spot. I've updated these to be `govuk-caption-l`

**Pull request merged**